editor/event_listener_line_edit.cpp

```diff
@@ -77,15 +77,15 @@ String EventListenerLineEdit::get_event_text(const Ref<InputEvent> &p_event, boo
 		}
 		if (key->get_physical_keycode() != Key::NONE) {
 			if (!text.is_empty()) {
-				text += " or ";
+				text += " " + RTR("or") + " ";
```

Comment:
Why RTR and not TTR? This is editor code.

Reply:
No idea, I only used it, because it was already used in this file.
I could change everything to TTR.